rmw_zenoh_cpp/src/detail/graph_cache.hpp

```diff
@@ -189,15 +189,6 @@ class GraphCache final
   /// Returns true if the entity is a publisher or client. False otherwise.
   static bool is_entity_pub(const liveliness::Entity & entity);
 
-  void set_querying_subscriber_callback(
```

Comment:
You'll also need to delete `querying_subs_cbs_` and update `graph_cache.cpp` to not iterate over `querying_subs_cbs_`

Reply:
Done in 9cc776bd8cc7fddfea8dec0ad703a22d3a6c96d3